v2/apiserver/internal/lib/restmachinery/endpoints.go

```diff
@@ -158,6 +158,9 @@ func WriteAPIResponse(
 	statusCode int,
 	response interface{},
 ) {
+	if statusCode == 0 {
```

Comment:
Curious on what provoked these changes?

Reply:
This seems to have broken in Go 1.17. It seems that previously, no status code was treated as 200. Does not seem to be the case anymore.